packages/vat-data/src/types.d.ts

```diff
@@ -8,13 +8,16 @@
 import type {
   InterfaceGuard,
   MapStore,
+  Pattern,
   SetStore,
   StoreOptions,
   WeakMapStore,
   WeakSetStore,
 } from '@agoric/store';
 
-type Baggage = MapStore<string, unknown>;
+export type { Pattern };
```

Comment:
This was needed because `Pattern=` had been hiding the missing type def.

The other exports in this file were needed after this started exporting.